docs/content/token/staking2/node-operators.mdx

```diff
@@ -64,67 +79,81 @@ The **node operator** should confirm that they have access to this capability.
 
 ## Stake New Tokens
 
-To stake new tokens, the node operator must use use transaction [../transactions/#staking-with-stakingproxynodestakerproxy](NO.05):
+The **node operator** can stake additional tokens on behalf of the **token holder**. 
 
-`stakingProxy/sh_stake_new_tokens.cdc`
+_Note: This transaction stakes additional tokens to the same node that was registered in the setup phase.
+It is currently not possible to stake to multiple nodes from the same account._
 
-Transaction arguments:
+To stake new tokens via the `NodeStakerProxy`, 
+the **node operator** can use the **Stake New Locked FLOW** ([NO.05](../transactions/#node-operator-no)) 
+transaction with the following arguments:
 
-| Argument   | Type   | Purpose |
-|------------|--------|---------|
-| **nodeID** | String | The unique identifier for the node. |
-| **amount** | UFix64 | The quantity of new FLOW tokens to add to the stake. |
+| Argument | Type | Purpose |
+|----------|------|---------|
+| **nodeID** | `String` | The unique identifier for the node. |
+| **amount** | `UFix64` | The number of FLOW tokens to stake. |
 
 ## Re-Stake Unstaked Tokens
 
-To re-stake previously unstaked tokens, the node operator must use use transaction [../transactions/#staking-with-stakingproxynodestakerproxy](NO.06):
+After tokens become unstaked, the **node operator** can choose to re-stake the unstaked tokens to the same node.
 
-`stakingProxy/sh_stake_unstaked_tokens.cdc`
+To staked unstaked tokens via the `NodeStakerProxy`, 
+the **node operator** can use the **Re-stake Unstaked FLOW** ([NO.06](../transactions/#node-operator-no)) 
+transaction with the following arguments:
 
-Transaction arguments:
-
-| Argument   | Type   | Purpose |
-|------------|--------|---------|
-| **nodeID** | String | The unique identifier for the node. |
-| **amount** | UFix64 | The quantity of previously unstaked FLOW tokens to add to the stake. |
+| Argument | Type | Purpose |
+|----------|------|---------|
+| **nodeID** | `String` | The unique identifier for the node. |
+| **amount** | `UFix64` | The number of FLOW tokens to stake. |
 
 ## Unstake Tokens
 
-To unstake **all previously staked** tokens, the node operator must use use transaction [../transactions/#staking-with-stakingproxynodestakerproxy](NO.07):
-
-`stakingProxy/sh_unstake_all.cdc`
+The **node operator** can submit a request to unstake the **node operator's** staked tokens at any time.
```

Comment:
```suggestion
The **node operator** can submit a request to unstake the **token holder's** staked tokens at any time.
```

Reply:
I actually put "token holder" on purpose, since they're doing the unstaking on behalf of the token holder and I didn't want there to be any confusion